plugins/amazonq/build.gradle.kts

```diff
@@ -23,15 +25,11 @@ tasks.jar {
     }
 }
 
-intellij {
-    plugins.set(
-        listOf(
-            project(":plugin-core")
-        )
-    )
-}
-
 dependencies {
+    intellijPlatform {
+        localPlugin(project(":plugin-core", "pluginZip"))
```

Comment:
what is the pluginZip used for?

Reply:
https://github.com/aws/aws-toolkit-jetbrains/pull/4524/files#diff-16d946c3e3b8d75616c236e9343f92d0dfb807ebbbcab7cf03037cdbc14693d7R90